From aad134067b2e1aee70b7799b75df4f73635bdd48 Mon Sep 17 00:00:00 2001
From: slin14 <shuicheng.lin@intel.com>
Date: Tue, 7 Aug 2018 23:25:34 +0800
Subject: [PATCH 13/15] commit f944ef677dc090e91b790ac54064d61d071edb5c Author:
 Shoaib Nasir <shoaib.nasir@windriver.com> Date:   Mon Sep 25 12:20:43 2017
 -0400

    Add ironic-uid-gid.patch to SPECS

Signed-off-by: slin14 <shuicheng.lin@intel.com>
---
 SPECS/setup.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/setup.spec b/SPECS/setup.spec
index 367a13b..974113e 100644
--- a/SPECS/setup.spec
+++ b/SPECS/setup.spec
@@ -30,6 +30,7 @@ Patch11: snmpd-fm-user-group.patch
 Patch12: remove-unused-default-groups.patch
 Patch13: add-fm-user-to-snmpd-group.patch
 Patch14: add-magnum-uid-gid.patch
+Patch15: add-ironic-uid-gid.patch
 
 %description
 The setup package contains a set of important system configuration and
@@ -52,6 +53,7 @@ setup files, such as passwd, group, and profile.
 %patch12 -p1
 %patch13 -p1
 %patch14 -p1
+%patch15 -p1
 
 ./shadowconvert.sh
 
-- 
2.7.4

